normalizer/docker/parser.py

```diff
@@ -41,6 +43,42 @@ def __init__(
 
     def __str__(self):
         return self.content
+    
+    def get_build_args_envs(self, build_args_envs): 
+        build_args = '';
+        for index, item in enumerate(build_args_envs):
+            build_args += dedent(
+            f"""\
+            ARG {item}
+            """
+        )
+        build_envs = '';
+        for index, item in enumerate(build_args_envs):
+            build_envs += dedent(
+            f"""\
+            ENV {item}=${item}
+            """
+        )
+        return (build_args, build_envs)
+
+    def add_build_args_envs(self, build_args_envs):
+        build_args, build_envs = self.get_build_args_envs(build_args_envs)
+        if len(build_args):
+            self._parser.content+=build_args
+        if len(build_args):
+            self._parser.content+=build_envs
+    
+    def insert_build_args_envs(self, build_args_envs):
```

Comment:
Shall we remove `add_build_args_envs` and use `insert_build_args_envs` for both dockerfile-existing and none-existing cases?

Reply:
done,  But we have to make sure that the initialized dockerfile must have the FROM keyword